operators/audio/audio_decoder.hpp

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "ocos.h"
+
+#include <list>
+#define DR_FLAC_IMPLEMENTATION
+#include "dr_flac.h"
+#define DR_MP3_IMPLEMENTATION 1
+#define DR_MP3_FLOAT_OUTPUT 1
+#include "dr_mp3.h"
+#define DR_WAV_IMPLEMENTATION
+#include "dr_wav.h"
+
+#include "string_utils.h"
+#include "string_tensor.h"
+
+struct KernelAudioDecoder : public BaseKernel {
+ public:
+  KernelAudioDecoder(const OrtApi& api, const OrtKernelInfo& info) : BaseKernel(api, info) {
+  }
+
+  enum {
+    DEFAULT_STREAM = 0,
+    WAV_STREAM,
+    MP3_STREAM,
+    FLAC_STREAM
+  };
+
+  int ReadStreamFormat(OrtKernelContext* context, const int64_t* p_data) {
+    int stream_format = DEFAULT_STREAM;
```

Comment:
is there anyway we can do this with size_t instead? (for consistency because we try to avoid using int because of overflow issues)

Reply:
this int is used only for enumuration, and no algorithmic operation with other value, so no worry for overflow.

Reply:
check line: 47, this format value needs to be changed, which cannot be supported by enum type.